docs/workflows/templates.md

```diff
@@ -0,0 +1,61 @@
+## Content Templates
+
+## Background
+
+Content Templates allow users to set a baseline of repository snapshots that help them control patching in predictable ways. In the future,
+users will be able to build images with a Content Template.
+
+### What is a Content Template?
+
+Content Templates are entities that can include snapshots from different Red Hat and custom repositories. 
+
+The content included in Content Templates is served at different distribution paths in pulp for Red Hat and custom since they are in different domains.
+
+For Red Hat content, the distribution path would be `/pulp/content/<red_hat_domain>/templates/<template_uuid>/`. For example, for this [Red Hat repository](https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/ansible/2/os/), 
+the content would live at `/pulp/content/<red_hat_domain>/templates/<template_uuid>/content/dist/layered/rhel8/x86_64/ansible/2/os/`.
+
+For custom content, the distribution path would be `/pulp/content/<custom_domain>/templates/<template_uuid>/<repository_uuid>/`.
+
+### What happens on our end when a Content Template is created/updated/deleted?
+
+After a repository has been snapshotted, a Content Template can be created with that snapshot. 
+
+On Content Template creation/update:
+
+1. The Content Template is created or updated and stored in our database with associations to the repositories that are included in the Content Template
+2. A task is started that first creates or updates the pulp distributions, then makes the necessary changes to the related entities in Candlepin
+
+On Content Template deletion:
+
+1. The Content Template is soft-deleted, and thus no longer visible to the user
+2. A task is started to permanently delete the Content Template in our database, its distributions in pulp, and its related entities in Candlepin
+
+### What happens in Candlepin when a Content Template is created/updated/deleted?
+
+The task kicked off on Content Template create/update/delete modifies any entities in Candlepin needed to represent the Content Template. 
+These entities are:
+
+* **Product** - contains the custom repositories included in the Content Template. Can be subscribed to by the owner. Currently we only create one custom product per org.
```

Comment:
```suggestion
* **Product** : contains the custom repositories included in the Content Template. Can be subscribed to by the owner. Currently we only create one custom product per org.
```
No dash and slash please. (Some American style guide does allow use of emdash, without spaces on either side, but its not really suitable for tech docs).